defaultValues.js

```diff
@@ -0,0 +1,5 @@
+DEFAULT_PORT = 8000;
+DEFAULT_ADRESS = 'localhost';
```

Comment:
Sorry for spellchecking, but it is `ADDRESS`.

Reply:
Shame on me.